wurst/event/DamageModify.wurst

```diff
@@ -0,0 +1,192 @@
+package DamageModify
```

Comment:
So, this is nice, but I'm not entirely happy with it.
For once it misses some documentation. From a `DamageModify` package I expect easy to use `modifyDamage` functions which I can use to modify any damageEvent.
Right now this looks to me as if I have to add a `ForDamageCallback` and check the Damage object passed to it.
Then I also have to manually destroy it.
Can you add some documentation with usage examples? Am I getting this wrong?

I would prefer at least some wrappers to easily manipulate any ongoing event. 

Also can you add some unit tests for this? (if there are missing natives, inform me)

Reply:
Btw, @Frotty, what kind of unit tests do you need? Because if you're unable to simulate wc3's units the same way the game does, there's nothing to test in damage based package.